android/modules/ui/src/java/ti/modules/titanium/ui/WindowProxy.java

```diff
@@ -164,6 +166,10 @@ protected void handleOpen(KrollDict options)
 	            topActivity.startActivity(intent);
 	        }
 	    }
+	    
+	    if (options.containsKey(TiC.PROPERTY_SUSTAINED_PERFORMANCE_MODE)) {
+	        setPropertyAndFire(TiC.PROPERTY_SUSTAINED_PERFORMANCE_MODE, options.getBoolean(TiC.PROPERTY_SUSTAINED_PERFORMANCE_MODE));
```

Comment:
I was afraid of adding it currently because that feature is only available in Android API 24 and not 23. We only currently support 23. But if I were to add it, it would be in the thread state methods in `TiBaseActivity`